src/Resources/config/services.xml

```diff
@@ -5,16 +5,13 @@
 
     <parameters>
         <parameter key="eight_points_guzzle.http_client.class">GuzzleHttp\Client</parameter>
-        <parameter key="eight_points_guzzle.middleware.class">GuzzleHttp\Middleware</parameter>
         <parameter key="eight_points_guzzle.formatter.class">GuzzleHttp\MessageFormatter</parameter>
 
         <parameter key="eight_points_guzzle.data_collector.class">EightPoints\Bundle\GuzzleBundle\DataCollector\HttpDataCollector</parameter>
         <parameter key="eight_points_guzzle.logger.class">EightPoints\Bundle\GuzzleBundle\Log\Logger</parameter>
-        <parameter key="eight_points_guzzle.middleware.log.class">EightPoints\Bundle\GuzzleBundle\Middleware\LogMiddleware</parameter>
 
-        <!-- Event MiddleWare Dispatcher class -->
+        <parameter key="eight_points_guzzle.middleware.log.class">EightPoints\Bundle\GuzzleBundle\Middleware\LogMiddleware</parameter>
         <parameter key="eight_points_guzzle.middleware.event_dispatcher.class">EightPoints\Bundle\GuzzleBundle\Middleware\EventDispatchMiddleware</parameter>
-
-        <parameter key="eight_points_guzzle.plugin.header.headers" type="collection" />
+        <parameter key="eight_points_guzzle.middleware.request_time.class">EightPoints\Bundle\GuzzleBundle\Middleware\RequestTimeMiddleware</parameter>
```

Comment:
We should also remove theses parameters see https://symfony.com/doc/current/best_practices/business-logic.html#service-no-class-parameter I can create an issue if you don't want to do it in this PR.

Reply:
@Neirda24 I will not do this change in this PR because it will break Backward Compatibility and we will be forced to release new major version only for this small change. And I think this article is more written for "own bundles" and not for "third-part bundles". I agree that "overriding a service by just changing its class name is a very rare use case" but we need to take a decision how to do and if we agree to remove them then probably we need to provide an alternative in Readme :slightly_smiling_face: @florianpreusner

Reply:
@gregurco : Then maybe do not remove these lines. Maybe they are used in some other projects (for any reasons). Regarding the article it is not only for making it "clean" it is for any service (third party or not) in order to avoid to flood the container with too many parameters that are not usefull.

Reply:
@Neirda24 I removed parameters that are not used by this bundle. Users will be able to upgrade without any errors. They still will be able to use them until they want that. This change don't create BC problem.

Reply:
hmmm. tricky question here... In my opinion the fact that the parameter exists it belongs to the "public API" even if it is not used by this library it doesn't mean someone that installed it doesn't use it...

Reply:
hm, my bad. I thought about rewriting them and not about using them. A moved them back but wrote a comment that they are deprecated. Thanks.

Reply:
no problem